### PR TITLE
Changes to resize entire view when keyboard appears

### PR DESCRIPTION
**Pull request by tomastiminskas**

### CODE REVIEW

File: sphinx/activity/main/activitymain/src/main/java/chat/sphinx/activitymain/MainActivity.kt [hunk 1]
```diff
@@ -80,7 +80,7 @@ internal class MainActivity: MotionLayoutNavigationActivity<
     override fun onCreate(savedInstanceState: Bundle?) {
         setTheme(R_common.style.AppPostLaunchTheme)
         super.onCreate(savedInstanceState)
-        WindowCompat.setDecorFitsSystemWindows(window, false)
```
Comment (05nelsonm) [inline sphinx/activity/main/activitymain/src/main/java/chat/sphinx/activitymain/MainActivity.kt:83]: This messes up the window insets for the entire app and must be false.

Comment (tomastiminskas) [reply sphinx/activity/main/activitymain/src/main/java/chat/sphinx/activitymain/MainActivity.kt:83]: I was testing and didn't see any issue setting that to "true". I think the `statusBarInsetHeight` and `navigationBarInsetHeight` will be always 0 with this configuration. Everything looks good on my end, even the fragments presented over the main fragment.

Comment (05nelsonm) [reply sphinx/activity/main/activitymain/src/main/java/chat/sphinx/activitymain/MainActivity.kt:83]: no_insets

Setting that to false removes transparent window insets

Comment (tomastiminskas) [reply sphinx/activity/main/activitymain/src/main/java/chat/sphinx/activitymain/MainActivity.kt:83]: I know. This is exactly what I described on the Kotlin tribe. Not sure if you read it. I discussed that with Paul and explained him that I couldn't make status and navigation bar transparent and view resizing when keyboard appears working at the same time. The only way I could prevent all the chat view to move up when keyboard is presented, is setting FitsSystemWindows to TRUE which results in bars becoming black. I think he prefers than than the chat header being out of the screen when keyboard is there. You may know a way to have transparent bars and adjustResize configuration working together, but I couldn't get that working easily. 

You can discuss with @pitoi which option to include in the next APK

**Comment by tomastiminskas** [comment]
Resolved on #410

**Pull request closed without merging**